examples/aws-ec2-pulumi/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "aws-ec2-pulumi",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "deploy": "go run ../../cmd/sst deploy",
+    "remove": "go run ../../cmd/sst remove",
```

Comment:
what is this pointing to?

Reply:
Should probably be removed but this lets you deploy using the local version of sst.